### PR TITLE
[VIVO-1846] i18n: Added i18n support for Capability Map control buttons

### PR DESCRIPTION
**Pull request by dofeldsc**
[JIRA Ticket](https://jira.lyrasis.org/browse/VIVO-1846)

This PR will add i18n support for the Capability Map control buttons

for description see [PR in VIVO Repository](https://github.com/vivo-project/VIVO/pull/175)

### CODE REVIEW

File: de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties [hunk 1]
```diff
@@ -582,6 +582,14 @@ vis_tools_note_three = Zurzeit speichern wir diese Modelle im Hauptspeicherberei
 vis_tools_note_four = Die Modelle werden bei jedem Neustart des Servers aktualisiert. Da dies auf der Produktionsinstanz  allgemein unpraktisch ist, können AdministratorInnen stattdessen den “Zwischenspeicher aktualisieren”-Link oben verwenden, um ohne einen Neustart eine Aktualisierung durchzuführen.
 capability_map=Forschungsnetz
 term_capitalized=Begriff
+pause=pause
```
Comment (awoods) [inline de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties:585]: Is "pause" the same in German and English?

Comment (gneissone) [reply de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties:585]: I'm not an expert, but seems likely 
Screen Shot 2020-06-26 at 9 06 49 AM

Comment (dofeldsc) [reply de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties:585]: yes :)

**Comment by dofeldsc** [comment]
all the "resume" translations need to be checked again
and the german "pause" will be changed to "pausieren" (thanks to Matthias)

**Pull request merged**